src/functional/markdown-full.spec.js

```diff
@@ -157,168 +162,160 @@ const listOfFileForEachCommit = {
   a0000015: ["untitled/script.md"],
   a0000016: ["packages/return-of-the-jedi/package.json"],
   a0000017: ["packages/return-of-the-jedi/package.json"],
+  a0000018: ["packages/return-of-the-jedi/package.json"],
 };
 
 const usersCache = {
-  "https://api.github.com/users/luke": {
-    body: {
-      login: "luke",
-      html_url: "https://github.com/luke",
-      name: "Luke Skywalker",
-    },
-  },
-  "https://api.github.com/users/princess-leia": {
-    body: {
-      login: "princess-leia",
-      html_url: "https://github.com/princess-leia",
-      name: "Princess Leia Organa",
-    },
-  },
-  "https://api.github.com/users/vader": {
-    body: {
-      login: "vader",
-      html_url: "https://github.com/vader",
-      name: "Darth Vader",
-    },
-  },
-  "https://api.github.com/users/gtarkin": {
-    body: {
-      login: "gtarkin",
-      html_url: "https://github.com/gtarkin",
-      name: "Governor Tarkin",
-    },
-  },
-  "https://api.github.com/users/han-solo": {
-    body: {
-      login: "han-solo",
-      html_url: "https://github.com/han-solo",
-      name: "Han Solo",
-    },
-  },
-  "https://api.github.com/users/chewbacca": {
-    body: {
-      login: "chewbacca",
-      html_url: "https://github.com/chewbacca",
-      name: "Chwebacca",
-    },
-  },
-  "https://api.github.com/users/rd-d2": {
-    body: {
-      login: "rd-d2",
-      html_url: "https://github.com/rd-d2",
-      name: "R2-D2",
-    },
-  },
-  "https://api.github.com/users/c-3po": {
-    body: {
-      login: "c-3po",
-      html_url: "https://github.com/c-3po",
-      name: "C-3PO",
-    },
-  },
-  "https://api.github.com/users/bot-user": {
-    body: {
-      login: "bot-user",
-      html_url: "https://github.com/bot-user",
-      name: "Bot User",
-    },
+  luke: {
+    login: "luke",
+    html_url: "https://github.com/luke",
+    name: "Luke Skywalker",
+  },
+  "princess-leia": {
+    login: "princess-leia",
+    html_url: "https://github.com/princess-leia",
+    name: "Princess Leia Organa",
+  },
+  vader: {
+    login: "vader",
+    html_url: "https://github.com/vader",
+    name: "Darth Vader",
+  },
+  gtarkin: {
+    login: "gtarkin",
+    html_url: "https://github.com/gtarkin",
+    name: "Governor Tarkin",
+  },
+  "han-solo": {
+    login: "han-solo",
+    html_url: "https://github.com/han-solo",
+    name: "Han Solo",
+  },
+  chewbacca: {
+    login: "chewbacca",
+    html_url: "https://github.com/chewbacca",
+    name: "Chwebacca",
+  },
+  "rd-d2": {
+    login: "rd-d2",
+    html_url: "https://github.com/rd-d2",
+    name: "R2-D2",
+  },
+  "c-3po": {
+    login: "c-3po",
+    html_url: "https://github.com/c-3po",
+    name: "C-3PO",
+  },
+  "bot-user": {
+    login: "bot-user",
+    html_url: "https://github.com/bot-user",
+    name: "Bot User",
   },
 };
 const issuesCache = {
-  "https://api.github.com/repos/embroider-build/github-changelog/issues/1": {
-    body: {
+  a0000001: [],
```

Comment:
again I'm surprised there is anything in here 🤔 why can't we just leave it empty for that commit sha?

Reply:
This is now a commit to PRs map